Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteTextureViewProxy.cpp

```diff
@@ -33,16 +33,14 @@
 
 namespace WebKit::WebGPU {
 
-RemoteTextureViewProxy::RemoteTextureViewProxy(RemoteTextureProxy& parent, ConvertToBackingContext& convertToBackingContext, WebGPUIdentifier identifier)
+RemoteTextureViewProxy::RemoteTextureViewProxy(RemoteGPUProxy& root, ConvertToBackingContext& convertToBackingContext, WebGPUIdentifier identifier)
     : m_backing(identifier)
     , m_convertToBackingContext(convertToBackingContext)
-    , m_parent(parent)
+    , m_root(root)
 {
 }
 
-RemoteTextureViewProxy::~RemoteTextureViewProxy()
```

Comment:
this change isn't really necessary in this PR, and will be un-done by https://github.com/WebKit/WebKit/pull/8856

Reply:
🆗